src/Statistics/Histogram.cpp

```diff
@@ -17,20 +17,32 @@
 namespace orbit_statistics {
 
 constexpr uint32_t kNumberOfBinsGridSize = 12;
+constexpr size_t kLargeNumberOfBins = 2048;  // 2^11
+constexpr uint32_t kVeryLargeDatasetThreshold = 10'000'000;
```

Comment:
Nit to all those constants. If they are like "arbitrary but large/small enough", state this in a comment. Otherwise, state how you came up with those. If not, future readers might wonder.